app/code/Magento/InventoryInStorePickupApi/etc/webapi.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<routes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Webapi:etc/webapi.xsd">
+    <route url="/V1/inventory/in-store-pickup/get-nearby-pickup-locations" method="GET">
```

Comment:
It would be a fairly uncommon practice given 3 parameters and `GET` request.
Not meaning that it's bad, but some additional ideas could be useful here.

Reply:
Agree. Changed to `POST`.